modules/apps/29-fee/keeper/genesis_test.go

```diff
@@ -18,9 +18,9 @@ func (suite *KeeperTestSuite) TestInitGenesis() {
 		uint64(1),
 	)
 	fee := types.Fee{
-		validCoins,
-		validCoins2,
-		validCoins3,
+		defaultReceiveFee,
```

Comment:
Was actually thinking we should rename the `TestXyz` variables in the ICA tests to be unexported and `defaultXyz`. I think it would look slightly cleaner like you've got here! I had made a note about it, can do it in a small review later this week 👍